src/main/java/com/microsoft/jenkins/azuread/AzureCachePool.java

```diff
@@ -27,16 +29,24 @@ public static AzureCachePool get(Azure.Authenticated azure) {
         return new AzureCachePool(azure);
     }
 
-    public Collection<String> getBelongingGroupsByOid(final String oid) {
+    public Collection<ActiveDirectoryGroup> getBelongingGroupsByOid(final String oid) {
         try {
-            Collection<String> result = belongingGroupsByOid.get(oid, new Callable<Collection<String>>() {
+            Collection<ActiveDirectoryGroup> result = belongingGroupsByOid.get(oid,
+                    new Callable<Collection<ActiveDirectoryGroup>>() {
                 @Override
-                public Collection<String> call() throws Exception {
+                public Collection<ActiveDirectoryGroup> call() throws Exception {
                     UserGetMemberGroupsParametersInner getMemberGroupsParam = new UserGetMemberGroupsParametersInner()
                             .withSecurityEnabledOnly(false);
                     List<String> groups = azure.activeDirectoryUsers().inner().getMemberGroups(oid,
```

Comment:
Please help rename the variable name here, it should be objectIds instead groups.

Reply:
Not sure what you mean here. Which variable are you talking about. If its `belongingGroupsByOid`, then i think its correct.